docs/fabric8-maven-plugin.adoc

```diff
@@ -0,0 +1,42 @@
+= Cube Integration Fabric8 Maven Plugin
```

Comment:
Shouldn't a chapter start with `==` ? Did you render the doc to see how it fits to the rest of the content?

I'm getting:

```
asciidoctor: ERROR: docs/fabric8-maven-plugin.adoc: line 1: only book doctypes can contain level 0 sections
```

Reply:
Also note, that just after that there is another section included called `fabric8` - shouldn't we merge those?